usr/lib/ep11_stdll/ep11_specific.h

```diff
@@ -43,6 +43,19 @@
 #define MAX_SIGN_STATE_BYTES        12288
 #define MAX_APQN                    256
 #define EP11_BLOB_WKID_OFFSET       32
+#define EP11_BLOB_BOOL_ATTR_OFFSET  52 /* second 32 bits of bool attr field */
```

Comment:
Just to make sure there's no typo: the 52 is correct here?

Reply:
Yes, 52 is the correct offset.